template/golang-middleware/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"handler/function"
+	//"github.com/openfaas-incubator/golang-http-template/template/golang-middleware/function"
+)
+
+func main() {
+	s := &http.Server{
+		Addr:           fmt.Sprintf(":%d", 8081),
+		ReadTimeout:    3 * time.Second,
+		WriteTimeout:   3 * time.Second,
+		MaxHeaderBytes: 1 << 20, // Max header of 1MB
+	}
+
+	http.HandleFunc("/", function.Handle)
```

Comment:
Do you want any buffering or additional middleware passing through here, or a direct connection?

Reply:
Nop I want to keep it simple